app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -123,6 +129,24 @@ def process_collection(collection, key)
         end
       end
 
+      def gather_data_for_this_region(arm_service)
+        results = []
+        @data[:resource_groups].each do |resource_group|
+          results << arm_service.list(resource_group[:name])
```

Comment:
@blomquisg - we have discussed this, @djberg96 estimates it is quite a bit of work.  I am guessing the separate PR wait until after the release?